### PR TITLE
Escape multiple $ signs in one line

### LINKED ISSUE #13004: Accidental MathJax rendering
Seems like the dollar sign triggered mathjax rendering under the section `#add-occupancy-pricing` in these lines

```
Trucks over 5000 lbs are charged an extra 5.00.Lighttrucksunder3000lbsaregivena2.00 discount. That rule is implemented with the following code:
```
image

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7bc5c533-a9ca-cc70-af20-21308e819082
* Version Independent ID: ef6af5d2-b640-0feb-d9d3-7aae0ca92264
* Content: [Use pattern matching features to extend data types](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/pattern-matching#add-occupancy-pricing)
* Content Source: [docs/csharp/tutorials/pattern-matching.md](https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/pattern-matching.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes #13004 

It looks to be the only line with multiple dollar signs.
And now I know how to do math in docs :).

### CODE REVIEW

File: docs/csharp/tutorials/pattern-matching.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ vehicle switch
 };
 ```
 
-The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra $5.00. Light trucks under 3000 lbs are given a $2.00 discount. That rule is implemented with the following code:
+The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra \$5.00. Light trucks under 3000 lbs are given a \$2.00 discount. That rule is implemented with the following code:
```
Comment (rpetrusha) [inline docs/csharp/tutorials/pattern-matching.md]:
```suggestion
The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra &#36;5.00. Light trucks under 3000 lbs are given a &#36;2.00 discount. That rule is implemented with the following code:
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This didn't correct the problem, @pkulikov:

image

Possibly HTML encoding (in my suggestion) or URL encoding (`%24`) will.

File: docs/csharp/tutorials/pattern-matching.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/tutorials/pattern-matching.md]: @rpetrusha thanks for the check! Let's try another way

File: docs/csharp/tutorials/pattern-matching.md [hunk 2]
```diff
@@ -193,7 +193,7 @@ vehicle switch
 };
 ```
 
-The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra $5.00. Light trucks under 3000 lbs are given a $2.00 discount. That rule is implemented with the following code:
+The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra &#36;5.00. Light trucks under 3000 lbs are given a &#36;2.00 discount. That rule is implemented with the following code:
```
Comment (rpetrusha) [inline docs/csharp/tutorials/pattern-matching.md]:
```suggestion
The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra %245.00. Light trucks under 3000 lbs are given a %242.00 discount. That rule is implemented with the following code:
```

**Review by rpetrusha — COMMENTED** [review]
That didn't work either, @pkulikov; the $ signs are interpreted as begin/end italics symbols. I don't have any degree of confidence that this change will solve the problem, but it's still worth trying.

File: docs/csharp/tutorials/pattern-matching.md (hunk 2 above)
Comment (pkulikov) [reply docs/csharp/tutorials/pattern-matching.md]: @rpetrusha thanks! Let's see; if that one fails, I'll rephrase the sentence.

Comment (BillWagner) [reply docs/csharp/tutorials/pattern-matching.md]: One other suggestion: We could split those paragraphs into multiple lines, so the `$` is on multiple source lines.

Comment (pkulikov) [reply docs/csharp/tutorials/pattern-matching.md]: @BillWagner thanks! it's simpler indeed. Would be the next option then. Next-next is to format as a list.

**Comment by rpetrusha** [comment]
This didn't work either, @pkulikov. I suspect it's a bug in the rendering engine. I'll open an issue for it. In the meantime, let's leave this PR open but tag it as blocked.

**Comment by pkulikov** [comment]
@rpetrusha can it be a configuration issue like mentioned here:
https://stackoverflow.com/questions/36030033/how-do-you-type-the-dollar-sign-when-you-have-mathjax-installed

Or it's not related to MathJax at all?

**Comment by pkulikov** [comment]
> This didn't work either

@rpetrusha meanwhile I've added a line break in the source file.

**Comment by BillWagner** [comment]
> I've added a line break in the source file.

*sigh* that didn't work either.

I'll ping some site folks internally.

**Comment by rpetrusha** [comment]
I don't think that it's related to MathJax.

**Comment by rpetrusha** [comment]
But actually, @pkulikov, it strikes me that the best thing to do here is to break the two items up into a list. That improves readability at the same time as it addresses the formatting issue.  One additional change: "Instead, they charge more based on the weight class of the trucks." --> "Instead, they charge based on the weight class of the trucks."

**Comment by pkulikov** [comment]
> One additional change:

@rpetrusha I've introduced the list formatting with a "more severe" change to the introductory sentence.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Your more severe change looks very good, @pkulikov, and with the list, the formatting now appears as expected. Thanks for your persistence and patience as we tried to fix this. I'll merge now.

**Pull request merged**